src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -45,7 +45,6 @@ public PlacedOrganelle()
     }
 
     public OrganelleDefinition Definition { get; set; }
-
```

Comment:
I don't see any reason to just remove line breaks from this file, especially as the rest of properties are separated by it.